src/common/CMakeLists.txt

```diff
@@ -67,9 +68,15 @@ else()
     set( LIBRARY_TYPE SHARED )
 endif()
 
+if( HAVE_CLOUDSC_GPU_SCC_FIELD )
+    list(APPEND CLOUDSC_COMMON_SOURCES
+        module/cloudsc_field_state_mod.F90
+    )
+endif()
+
 ecbuild_add_library( TARGET cloudsc-common-lib
     TYPE ${LIBRARY_TYPE}
-    DEFINITIONS ${CLOUDSC_DEFINITIONS}
+    DEFINITIONS ${CLOUDSC_DEFINITIONS} $<${HAVE_CLOUDSC_GPU_SCC_FIELD}:CLOUDSC_GPU_SCC_FIELD>
```

Comment:
This should be redundant if added to the CLOUDSC_DEFINITIONS in the root CMakeLists.txt